README.md

```diff
@@ -288,6 +288,37 @@ public class FailoverSampleApp2 {
   }
 }
 ```
+
+### Plugin Manager
+
+<div style="text-align:center"><img src="./docs/files/images/plugin_manager_diagram.png" /></div>
+
+Figure shows a simplified workflow of the plugin manager. Starting at the top, when a new query is executed by the application, it is passed into the plugin manager. From the  plugin manager, the query is passed in order of plugins loaded, in this figure, `Custom Plugin A` to `Custom Plugin B` and finally to `Default Plugin` which executes the query and returns the result sets back up the chain.
```

Comment:
Not only a query goes through the plugin manager.
I think this section should be called Connection Plugin Manager

Reply:
Plugins execute Jdbc methods.